doc/Makefile

```diff
@@ -100,3 +100,6 @@ download-data:
 optipng:
 	find _build auto_examples */generated -name '*.png' -print0 \
 	  | xargs -0 -n 1 -P 4 optipng -o10
+
+dist: html latexpdf
+	cp _build/latex/user_guide.pdf _build/html/stable/_downloads/scikit-learn-docs.pdf
```

Comment:
I am worried that people might not see this and thus not use it.

How do we address this? First the 'help' target needs to be modified, but maybe there are instructions to be added somewhere else?